README.md

```diff
@@ -40,6 +40,10 @@ interface Options {
   theme?: Theme;
   /** Render with a framing window, defaults to false */
   window?: boolean;
+  /** Idle time limit in milliseconds */
+  idle?: number,
```

Comment:
In the PR for load-asciinema, the unit is second:
https://github.com/marionebl/load-asciicast/pull/2/files

In this PR, because other options are already using millisecond, I also use it for `idle`.